### PR TITLE
Release changes

### PR DESCRIPTION
**Pull request by ckm007**
Automated PR for 1.2.0.1-B2 release.

### CODE REVIEW

File: kernel/kernel-idgenerator-machineid/pom.xml [hunk 1]
```diff
@@ -217,11 +217,11 @@
 		<aspectjweaver.version>1.8.12</aspectjweaver.version>
 		<micrometer.core.version>1.4.2</micrometer.core.version>
 		<micrometer.registry.prometheus.version>1.4.2</micrometer.registry.prometheus.version>
-		<kernel.core.version>1.2.0.1-SNAPSHOT</kernel.core.version>
-		<kernel.auth-adapter.version>1.2.0.1-SNAPSHOT</kernel.auth-adapter.version>
-		<kernel.applicant-type.version>1.2.0.1-SNAPSHOT</kernel.applicant-type.version>
-		 <kernel.dataaccess-hibernate.version>1.2.0.1-SNAPSHOT</kernel.dataaccess-hibernate.version>
-		<kernel.logger.version>1.2.0.1-SNAPSHOT</kernel.logger.version>
+		<kernel.core.version>1.2.0.1-B2</kernel.core.version>
+		<kernel.auth-adapter.version>1.2.0.1-B2</kernel.auth-adapter.version>
```
Comment (ckm007) [inline kernel/kernel-idgenerator-machineid/pom.xml]: shall this point to latest?

File: kernel/kernel-idgenerator-prid/pom.xml [hunk 2]
```diff
@@ -217,15 +217,15 @@
...
-		<kernel.core.version>1.2.0.1-SNAPSHOT</kernel.core.version>
-		<kernel.dataaccess-hibernate.version>1.2.0.1-SNAPSHOT</kernel.dataaccess-hibernate.version>
-		<kernel.logger.version>1.2.0.1-SNAPSHOT</kernel.logger.version>
-		<kernel.idvalidator-prid.version>1.2.0.1-SNAPSHOT</kernel.idvalidator-prid.version>
-		<kernel-keymanager-service.version>1.2.0.1-SNAPSHOT</kernel-keymanager-service.version>
+		<kernel.core.version>1.2.0.1-B2</kernel.core.version>
+		<kernel.dataaccess-hibernate.version>1.2.0.1-B2</kernel.dataaccess-hibernate.version>
+		<kernel.logger.version>1.2.0.1-B2</kernel.logger.version>
+		<kernel.idvalidator-prid.version>1.2.0.1-B2</kernel.idvalidator-prid.version>
+		<kernel-keymanager-service.version>1.2.0.1-B2</kernel-keymanager-service.version>
```
Comment (ckm007) [inline kernel/kernel-idgenerator-prid/pom.xml]: will this be latest?

File: kernel/kernel-idgenerator-tokenid/pom.xml [hunk 3]
```diff
@@ -218,9 +218,9 @@
 		<aspectjweaver.version>1.8.12</aspectjweaver.version>
 		<micrometer.core.version>1.4.2</micrometer.core.version>
 		<micrometer.registry.prometheus.version>1.4.2</micrometer.registry.prometheus.version>
-		<kernel.core.version>1.2.0.1-SNAPSHOT</kernel.core.version>
-		<kernel.dataaccess-hibernate.version>1.2.0.1-SNAPSHOT</kernel.dataaccess-hibernate.version>
-		<kernel-keymanager-service.version>1.2.0.1-SNAPSHOT</kernel-keymanager-service.version>
+		<kernel.core.version>1.2.0.1-B2</kernel.core.version>
+		<kernel.dataaccess-hibernate.version>1.2.0.1-B2</kernel.dataaccess-hibernate.version>
+		<kernel-keymanager-service.version>1.2.0.1-B2</kernel-keymanager-service.version>
```
Comment (ckm007) [inline kernel/kernel-idgenerator-tokenid/pom.xml]: will this be latest?

File: kernel/kernel-idgenerator-vid/pom.xml [hunk 4]
```diff
@@ -216,9 +216,9 @@
 		<aspectjweaver.version>1.8.12</aspectjweaver.version>
 		<micrometer.core.version>1.4.2</micrometer.core.version>
 		<micrometer.registry.prometheus.version>1.4.2</micrometer.registry.prometheus.version>
-		<kernel.core.version>1.2.0.1-SNAPSHOT</kernel.core.version>
-		<kernel.idvalidator-vid.version>1.2.0.1-SNAPSHOT</kernel.idvalidator-vid.version>
-		<kernel-keymanager-service.version>1.2.0.1-SNAPSHOT</kernel-keymanager-service.version>
+		<kernel.core.version>1.2.0.1-B2</kernel.core.version>
+		<kernel.idvalidator-vid.version>1.2.0.1-B2</kernel.idvalidator-vid.version>
+		<kernel-keymanager-service.version>1.2.0.1-B2</kernel-keymanager-service.version>
```
Comment (ckm007) [inline kernel/kernel-idgenerator-vid/pom.xml]: this should be latest?

File: kernel/kernel-notification-service/pom.xml [hunk 5]
```diff
@@ -221,11 +221,11 @@
 		<aspectjweaver.version>1.8.12</aspectjweaver.version>
 		<micrometer.core.version>1.4.2</micrometer.core.version>
 		<micrometer.registry.prometheus.version>1.4.2</micrometer.registry.prometheus.version>
-		<kernel.core.version>1.2.0.1-SNAPSHOT</kernel.core.version>
-		<kernel.auth-adapter.version>1.2.0.1-SNAPSHOT</kernel.auth-adapter.version>
-		<kernel.applicant-type.version>1.2.0.1-SNAPSHOT</kernel.applicant-type.version>
-		<kernel.dataaccess-hibernate.version>1.2.0.1-SNAPSHOT</kernel.dataaccess-hibernate.version>
-		<kernel.logger.version>1.2.0.1-SNAPSHOT</kernel.logger.version>
+		<kernel.core.version>1.2.0.1-B2</kernel.core.version>
+		<kernel.auth-adapter.version>1.2.0.1-B2</kernel.auth-adapter.version>
```
Comment (ckm007) [inline kernel/kernel-notification-service/pom.xml]: this should be latest?

File: kernel/kernel-notification-service/pom.xml [hunk 6]
```diff
@@ -491,7 +491,7 @@
 				<dependency>
 					<groupId>io.mosip.kernel</groupId>
 					<artifactId>kernel-smsserviceprovider-msg91</artifactId>
-					<version>1.2.0.1-SNAPSHOT-rc1</version>
+					<version>1.2.0.1-B2-rc1</version>
```
Comment (ckm007) [inline kernel/kernel-notification-service/pom.xml]: 1.2.0.1-B2-rc1  --> 1.2.0.1-B2

File: kernel/kernel-otpmanager-service/pom.xml [hunk 7]
```diff
@@ -219,10 +219,10 @@
 		<aspectjweaver.version>1.8.12</aspectjweaver.version>
 		<micrometer.core.version>1.4.2</micrometer.core.version>
 		<micrometer.registry.prometheus.version>1.4.2</micrometer.registry.prometheus.version>
-		<kernel.core.version>1.2.0.1-SNAPSHOT</kernel.core.version>
-		<kernel.auth-adapter.version>1.2.0.1-SNAPSHOT</kernel.auth-adapter.version>
-		<kernel.dataaccess-hibernate.version>1.2.0.1-SNAPSHOT</kernel.dataaccess-hibernate.version>
-		<kernel.logger.version>1.2.0.1-SNAPSHOT</kernel.logger.version>
+		<kernel.core.version>1.2.0.1-B2</kernel.core.version>
+		<kernel.auth-adapter.version>1.2.0.1-B2</kernel.auth-adapter.version>
```
Comment (ckm007) [inline kernel/kernel-otpmanager-service/pom.xml]: latest?

File: kernel/kernel-pridgenerator-service/pom.xml [hunk 8]
```diff
@@ -215,16 +215,16 @@
 		<aspectjweaver.version>1.8.12</aspectjweaver.version>
 		<micrometer.core.version>1.4.2</micrometer.core.version>
 		<micrometer.registry.prometheus.version>1.4.2</micrometer.registry.prometheus.version>
-		<kernel.core.version>1.2.0.1-SNAPSHOT</kernel.core.version>
-		<kernel.auth-adapter.version>1.2.0.1-SNAPSHOT</kernel.auth-adapter.version>
-		<kernel.templatemanager-velocity.version>1.2.0.1-SNAPSHOT</kernel.templatemanager-velocity.version>
-		<kernel.logger.version>1.2.0.1-SNAPSHOT</kernel.logger.version>
-		<kernel.prid-generator.version>1.2.0.1-SNAPSHOT</kernel.prid-generator.version>
+		<kernel.core.version>1.2.0.1-B2</kernel.core.version>
+		<kernel.auth-adapter.version>1.2.0.1-B2</kernel.auth-adapter.version>
```
Comment (ckm007) [inline kernel/kernel-pridgenerator-service/pom.xml]: latest

File: kernel/kernel-ridgenerator-service/pom.xml [hunk 9]
```diff
@@ -117,10 +117,10 @@
 		<aspectjweaver.version>1.8.12</aspectjweaver.version>
 		<micrometer.core.version>1.4.2</micrometer.core.version>
 		<micrometer.registry.prometheus.version>1.4.2</micrometer.registry.prometheus.version>
-		<kernel.core.version>1.2.0.1-SNAPSHOT</kernel.core.version>
-		<kernel.auth-adapter.version>1.2.0.1-SNAPSHOT</kernel.auth-adapter.version>
-		<kernel.dataaccess-hibernate.version>1.2.0.1-SNAPSHOT</kernel.dataaccess-hibernate.version>
-		<kernel.logger.version>1.2.0.1-SNAPSHOT</kernel.logger.version>
+		<kernel.core.version>1.2.0.1-B2</kernel.core.version>
+		<kernel.auth-adapter.version>1.2.0.1-B2</kernel.auth-adapter.version>
```
Comment (ckm007) [inline kernel/kernel-ridgenerator-service/pom.xml]: latest ?

File: kernel/kernel-notification-service/pom.xml (hunk 5 above)
Comment (ckm007) [reply kernel/kernel-notification-service/pom.xml]: here it is mentioned in profile.

File: kernel/kernel-otpmanager-service/pom.xml (hunk 7 above)
Comment (ckm007) [reply kernel/kernel-otpmanager-service/pom.xml]: mentioned in profile

File: kernel/kernel-ridgenerator-service/pom.xml (hunk 9 above)
Comment (ckm007) [reply kernel/kernel-ridgenerator-service/pom.xml]: mentioned in profiles

File: kernel/kernel-idgenerator-prid/pom.xml (hunk 2 above)
Comment (ckm007) [reply kernel/kernel-idgenerator-prid/pom.xml]: MOSIP-30716

File: kernel/kernel-idgenerator-vid/pom.xml (hunk 4 above)
Comment (ckm007) [reply kernel/kernel-idgenerator-vid/pom.xml]: MOSIP-30716

File: kernel/kernel-notification-service/pom.xml (hunk 6 above)
Comment (ckm007) [reply kernel/kernel-notification-service/pom.xml]: MOSIP-30716

File: kernel/kernel-idgenerator-tokenid/pom.xml (hunk 3 above)
Comment (ckm007) [reply kernel/kernel-idgenerator-tokenid/pom.xml]: MOSIP-30716

File: kernel/kernel-otpmanager-service/pom.xml (hunk 7 above)
Comment (ckm007) [reply kernel/kernel-otpmanager-service/pom.xml]: MOSIP-30716

File: kernel/kernel-pridgenerator-service/pom.xml (hunk 8 above)
Comment (ckm007) [reply kernel/kernel-pridgenerator-service/pom.xml]: MOSIP-30716

File: kernel/kernel-ridgenerator-service/pom.xml (hunk 9 above)
Comment (ckm007) [reply kernel/kernel-ridgenerator-service/pom.xml]: MOSIP-30716

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**